apps/backoffice-v2/src/pages/MerchantMonitoring/schemas.ts

```diff
@@ -125,6 +126,6 @@ export const MerchantMonitoringSearchSchema = BaseSearchSchema.extend({
       ],
     )
     .catch('All'),
-  from: z.string().date().optional(),
-  to: z.string().date().optional(),
+  from: z.string().date().default(dayjs().subtract(30, 'day').format('YYYY-MM-DD')),
+  to: z.string().date().default(dayjs().format('YYYY-MM-DD')),
```

Comment:
Prefer catch to default for search query params. This way it covers both invalid input and the default state.